docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+services:
+  toolhunt_backend:
+    container_name: toolhunt_backend
```

Comment:
nit: you can leave the `toolhunt` part out, as it's already contained in the name (run `docker ps` to see what I mean). You could go for something shorter like `web` or `flask-web`.